src/site/content/en/fast/lazy-loading-video/index.md

```diff
@@ -0,0 +1,170 @@
+which lets you specify a placeholder to occupy the `<video>` element's space
+until the video is lazy-loaded. As with our [`<img>` lazy-loading examples](/lazy-loading-images/),
+ we stash the video URL in the `data-src` attribute on each `<source>`
+element. From there, we'll use some JavaScript similar to the earlier
+Intersection Observer-based image lazy-loading examples:
+
+```javascript
+document.addEventListener("DOMContentLoaded", function() {
+  var lazyVideos = [].slice.call(document.querySelectorAll("video.lazy"));
+
+  if ("IntersectionObserver" in window) {
+    var lazyVideoObserver = new IntersectionObserver(function(entries, observer) {
+      entries.forEach(function(video) {
+        if (video.isIntersecting) {
+          for (var source in video.target.children) {
+            var videoSource = video.target.children[source];
+            if (typeof videoSource.tagName === "string" && videoSource.tagName === "SOURCE") {
+              videoSource.src = videoSource.dataset.src;
+            }
+          }
+
+          video.target.load();
+          video.target.classList.remove("lazy");
+          lazyVideoObserver.unobserve(video.target);
+        }
+      });
+    });
+
+    lazyVideos.forEach(function(lazyVideo) {
+      lazyVideoObserver.observe(lazyVideo);
+    });
+  }
+});
+```
+
+When you lazy-load a `<video>` element, you need to iterate through all of the child
+`<source>` elements and flip their `data-src` attributes to `src` attributes. Once
+you've done that, you need to trigger loading of the video by calling the
+element's `load` method, after which the media will begin playing automatically
+per the `autoplay` attribute.
+
+Using this method, you have a video solution that emulates animated GIF behavior,
+but doesn't incur the same intensive data usage as animated GIFs do,
+and you can lazy-load that content.
+
+## lazy-loading libraries {: #libraries }
+
+The following libraries can help you to lazy-load video:
+
+- [lozad.js](https://github.com/ApoorvSaxena/lozad.js) is a super lightweight
+option that uses Intersection Observer only. As such, it's highly performant,
+but will need to be polyfilled before you can use it on older browsers.
+- [yall.js](https://github.com/malchata/yall.js) is a library I wrote that uses
+Intersection Observer and falls back to event handlers. It's compatible with IE11
+and major browsers.
+- If you're seeking a React-specific lazy-loading library, you might consider
+[react-lazyload](https://github.com/jasonslyvia/react-lazyload). While it
+doesn't use Intersection Observer, it _does_ provide a familiar method of lazy
+loading images for those accustomed to developing applications with React.
+
+Each of these lazy-loading libraries is well documented, with plenty of markup
+patterns for your various lazy-loading endeavors. If you're not one to tinker,
+grab a library and go. It will take the least amount of effort.
```

Comment:
```suggestion
Each of these lazy-loading libraries is well documented, with plenty of markup
patterns for your various lazy-loading endeavors.
```

Reply:
I think there's a variety of reasons why someone would opt for using a library and calling out only the "least amount of effort" can sound a bit condescending.